WarehouseConfiguration warehouse_configuration
RobotConfiguration robot_configuration
---
bool success
